ga4gh/serverconfig.py

```diff
@@ -40,6 +40,8 @@ class DevelopmentConfig(BaseConfig):
     """
     DATA_SOURCE = "ga4gh-example-data"
     DEBUG = True
+    REQUEST_VALIDATION = True
+    RESPONSE_VALIDATION = True
```

Comment:
Not sure we want response validation turned on by default, as this is quite a big cost (and unnecessary really).

I would say set REQUEST_VALIDATION = True in BaseConfig and leave it at that. There's no good reason to have request validation turned off I think.